prometheus-to-sd/main.go

```diff
@@ -153,7 +163,7 @@ func readAndPushDataToStackdriver(stackdriverService *v3.Service, gceConf *confi
 		if strings.HasPrefix(commonConfig.GceConfig.MetricsPrefix, customMetricsPrefix) {
 			metricDescriptorCache.UpdateMetricDescriptors(metrics, sourceConfig.Whitelisted)
 		}
-		ts := translator.TranslatePrometheusToStackdriver(commonConfig, sourceConfig.Whitelisted, metrics, metricDescriptorCache)
+		ts := translator.TranslatePrometheusToStackdriver(commonConfig, sourceConfig.Whitelisted, gaugeToCumulativeWhitelistedList, metrics, metricDescriptorCache)
```

Comment:
The same, it doesn't compile and we you don't need (it's a part of common config)

Reply:
Done